unit-tests/py/rspy/devices.py

```diff
@@ -518,23 +499,23 @@ def enable_only( serial_numbers, recycle = False, timeout = MAX_ENUMERATION_TIME
                     re-enabling
     :param timeout: The maximum seconds to wait to make sure the devices are indeed online
     """
-    if acroname:
+    if relay and relay.has_relay():
```

Comment:
If we have a relay, then there's no need for `has_relay()` -- this method should be removed

Reply:
I see, in the original code we'd have an instance of usb_relay even if we didn't have a hub connected, but now if we don't have a hub, we won't have an instance (it will be None) so it's not needed now, I'll remove it